package.json

```diff
@@ -58,5 +74,11 @@
   },
   "peerDependencies": {
     "tslib": "^2.1.0"
+  },
+  "jest": {
+    "roots": [
+      "src/bundles",
+      "src/tabs"
+    ]
```

Comment:
Wanna add testing capability to our generate template app as well? Might be a good idea idk.